MovieApp/SupportingFiles/Extensions/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+extension UIViewController {
+
+    func styleForTabBar(_ title: String, _ image: UIImage?, _ selectedImage: UIImage?, _ font: UIFont?) {
```

Comment:
Do not use _ in functions like this since it is not clear what argument is for what. For example, you have two UIImage arguments.